loader.js

```diff
@@ -9,5 +12,38 @@ module.exports = function(source) {
 	if (!preFlightCheck) {
 		throw new Error(pluginErrors.preflightError);
 	}
-	return 'module.exports = {};';
+
+	var ffprobeResponse = spawnSync('ffprobe', ['-v','quiet', '-print_format', 'json', '-show_format', this.resourcePath]);
```

Comment:
Really should do this work as async. Should let other loaders do any work they can instead of getting blocked by our IO needs.